Errors with debug colors

Hello! 👋 

I've seeing the following exceptions when using [ansi-econsole](https://github.com/mihnita/ansi-econsole) with the theme:

```
java.lang.ExceptionInInitializerError
	at java.base/java.lang.J9VMInternals.ensureError(J9VMInternals.java:193)
	at java.base/java.lang.J9VMInternals.recordInitializationFailure(J9VMInternals.java:182)
	at mnita.ansiconsole.participants.AnsiConsoleStyleListener.<init>(AnsiConsoleStyleListener.java:44)
	at mnita.ansiconsole.participants.AnsiConsolePageParticipant.init(AnsiConsolePageParticipant.java:45)
	at org.eclipse.ui.internal.console.ConsoleView$3.run(ConsoleView.java:343)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.ui.internal.console.ConsoleView.doCreatePage(ConsoleView.java:340)
	at org.eclipse.ui.part.PageBookView.createPage(PageBookView.java:369)
	at org.eclipse.ui.part.PageBookView.partActivated(PageBookView.java:696)
	at org.eclipse.ui.internal.console.ConsoleView.lambda$0(ConsoleView.java:412)
	at org.eclipse.ui.internal.console.ConsoleView$$Lambda$918/0000000009018220.run(Unknown Source)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:40)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:185)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4124)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3791)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1158)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1047)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:658)
	at org.eclipse.ui.internal.Workbench$$Lambda$120/0000000000000000.run(Unknown Source)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:557)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:154)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:150)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:401)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:657)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:594)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1447)
Caused by: java.lang.IllegalArgumentException: Argument not valid
	at org.eclipse.swt.SWT.error(SWT.java:4704)
	at org.eclipse.swt.SWT.error(SWT.java:4638)
	at org.eclipse.swt.SWT.error(SWT.java:4609)
	at org.eclipse.swt.graphics.RGB.<init>(RGB.java:80)
	at mnita.ansiconsole.preferences.AnsiConsolePreferenceUtils.colorFromStringRgb(AnsiConsolePreferenceUtils.java:72)
	at mnita.ansiconsole.preferences.AnsiConsolePreferenceUtils.refresh(AnsiConsolePreferenceUtils.java:151)
	at mnita.ansiconsole.preferences.AnsiConsolePreferenceUtils.<clinit>(AnsiConsolePreferenceUtils.java:44)
	... 35 more
```

That plugin is unusuccessfully trying to parse one of the debug colors here:
https://github.com/mihnita/ansi-econsole/blob/e962dbf1fa36e7a7e55b3be83ef0b0a4ab487bf7/AnsiConsole/src/mnita/ansiconsole/preferences/AnsiConsolePreferenceUtils.java#L150

Looking at the theme's CSS style sheet, I noticed that there were spaces in some color values:
https://github.com/AObuchow/Eclipse-Modern-Dark-Theme/blob/b2bf890947dd3a517f214cd30f1efba589c0570b/com.aobuchow.themes.moderndark/css/preference_styles.css#L105

I'm guessing this is what is causing the problem. :)

--- a/com.aobuchow.themes.moderndark/css/preference_styles.css
+++ b/com.aobuchow.themes.moderndark/css/preference_styles.css
@@ -102,8 +102,8 @@ IEclipsePreferences#org-eclipse-debug-ui:org-eclipse-ui-themes-modern-dark-debug
 		'org.eclipse.debug.ui.changedDebugElement=255,128,128'
 		'org.eclipse.debug.ui.consoleBackground=0,0,0'
 		'org.eclipse.debug.ui.outColor=255,255,255'
-		'org.eclipse.debug.ui.errorColor=255, 0, 0'
-		'org.eclipse.debug.ui.inColor=115, 13, 90'
+		'org.eclipse.debug.ui.errorColor=255,0,0'
+		'org.eclipse.debug.ui.inColor=115,13,90'
 }
 
 /* ############################## PDE ############################## */

